app/assets/stylesheets/application.scss

```diff
@@ -193,14 +194,20 @@ body {
 .btn-google {
   color: white;
   background-color: #ea4335;
+  font-size: 80%;
+  border-radius: 5rem;
+  letter-spacing: 0.1rem;
+  font-weight: bold;
+  padding: 1rem;
+  transition: all 0.2s;
```

Comment:
This is the only thing I actually did in this file... I just copied the CSS over from the facebook button.

Reply:
looking good